File: EstCondicional/Ejercicio31.py
#Declarar Variables
edad:int
mensaje:str
#Datos de Entrada
edad=int(input("Ingrese su edad:"))
#Proceso y Datos de Salida
if edad>=18:
  mensaje=(f"Puede votar porque su edad es {edad}")
else:
  mensaje=(f"No puede votar porque su edad es {edad}")
#Datos de Salida
print(mensaje)

File: EstCondicional/Ejercicio31.java

import java.util.Scanner;

public class Ejercicio31 {
    static Scanner lt=new Scanner(System.in);
    public static void main(String[] args) {
        //Declarar Variables
        int edad;
        String mensaje;
        //Datos de entrada
        System.out.println("Ingrese la edad:");
        edad=lt.nextInt();
        //Proceso
        if (edad>=18) {
            mensaje="Puede votar porque su edad es "+edad;
        } else {
            mensaje="No puede votar porque su edad es "+edad;
        }
        //Datos de Salida
        System.out.println(mensaje);        
    }
}
